terra-jupyter-dev-base/Dockerfile

```diff
@@ -0,0 +1,163 @@
+FROM ubuntu:20.04
+
+USER root
+
+# The welder uid is consistent with the Welder docker definition here:
+# https://github.com/DataBiosphere/welder/blob/master/project/Settings.scala
+# Adding welder-user to the Jupyter container isn't strictly required, but it makes welder-added
+# files display nicer when viewed in a terminal.
+ENV DEBIAN_FRONTEND=noninteractive \
+    LC_ALL=en_US.UTF-8 \
+    USER=jupyter \
+    WELDER_USER=welder-user \
+    WELDER_UID=1001 \
+    # ensure this matches c.NotebookApp.port in jupyter_notebook_config.py
+    JUPYTER_PORT=8000 \
+    JUPYTER_HOME=/etc/jupyter \
+    CONDA_AUTO_UPDATE_CONDA=false \
+    CONDA_DIR=/opt/conda
+ENV HOME=/home/$USER
+    # When using PIP_USER=true packages are installed into Python site.USER_BASE, which is '/home/jupyter' for this system.
+    # Append '/home/jupyter/.local/bin' to PATH
+    # pip docs: https://pip.pypa.io/en/stable/reference/pip_install/#cmdoption-user
+ENV PATH="${PATH}:${CONDA_DIR}/bin:${HOME}/.local/bin:${HOME}/packages/bin"
+
+# Users
+RUN useradd -m -s /bin/bash $USER \
+ && usermod -g users $USER \
+ && useradd -m -s /bin/bash -N -u $WELDER_UID $WELDER_USER \
+# Prerequisites
+ && apt-get update && apt-get install -yq --no-install-recommends \
+    sudo \
+ && sudo -i \
+    echo "deb http://security.ubuntu.com/ubuntu/ bionic main" >> /etc/apt/sources.list \
```

Comment:
`bionic` is code for 18.04. Is this supposed to be `Focal Fossa`?

Reply:
Good point: I am not sure about this.  I copied it directly from `terra-jupyter-base`. I'm honestly not sure what the base operating system actually is in `terra-jupyter-base`, since it uses a google deep learning base image `gcr.io/deeplearning-platform-release/tf-gpu.2-7`, and I can't seem to find the details.

The only thing I know is that, if I remove these steps from the build, it doesn't work.  These commands seem to be necessary for successful install of `libexempi3` and `libv8-3.14-dev` (which in turn are only needed in order to install the python packages `firecloud` and `terra-notebook-utils`)

Reply:
Oh, I didn't mean to delete these. I mean for 20.04, shouldn't there be a new name other than `bionic`?

Reply:
Right, yeah, you could be right.  I'm not sure.  Somehow this works as is.  Is it maybe a workaround to install `bionic` packages in later ubuntu releases?  Or maybe it does need to be updated from `bionic` to `focal fossa` here.  I would need to ask the person who wrote this part :)

Reply:
I believe it's now `focal` replacing `bionic` for 20.04.